### PR TITLE
PER-240 created a queue message handler

### PR DESCRIPTION
**Pull request by ChadRehmKineticData**
The handler can preform all CRUD operations available to Queue Message endpoint.

### CODE REVIEW

**Comment by ChadRehmKineticData** [comment]
This PR can be squashed.

File: handlers/azure_queue_messages_api_v1/handler/init.rb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+    @tenent_id = @info_values["tenent_id"]
+    @client_id = @info_values["client_id"]
+    @client_secret = @info_values["client_secret"]
+    
+    @headers = @parameters["headers"].to_s.empty? ? {} : JSON.parse(@parameters["headers"])
+    @body = @parameters["body"]
+    @method = (@parameters["method"] || :get).downcase.to_sym
+    @storage_account = @parameters["storage_account"]
+    puts @parameters
```
Comment (jackboespflug) [inline handlers/azure_queue_messages_api_v1/handler/init.rb:34]: Remove this. I assume you had it in here for troubleshooting during development.

File: handlers/azure_queue_messages_api_v1/handler/init.rb [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          "client_id" => @client_id,
+          "client_secret" => @client_secret,
+          "grant_type" => "client_credentials",
+          "scope" => "https://#{@storage_account}.queue.core.windows.net/.default"
+        }, \
+        headers: {:content_type => 'application/x-www-form-urlencoded', :accept => @accept}
+       
+      # Parse token from request
+      puts "Access token retrieved." if @debug_logging_enabled
+      token = JSON.parse(auth_response.body)['access_token']
```
Comment (jackboespflug) [inline handlers/azure_queue_messages_api_v1/handler/init.rb:70]: I don't understand why you're parsing the response body as JSON when telling the server you only accept XML. I assume the server doesn't respect the accept header and only returns JSON. Probably best just to remove the accept header from the request.

**Review by jackboespflug — CHANGES_REQUESTED** [review]

File: handlers/azure_queue_messages_api_v1/handler/init.rb (hunk 2 above)
Comment (ChadRehmKineticData) [reply handlers/azure_queue_messages_api_v1/handler/init.rb:69]: Removed

**Review by jackboespflug — APPROVED** [review]

**Pull request merged**